pom.xml

```diff
@@ -138,6 +138,7 @@
     <jacoco.version>0.8.5</jacoco.version>
     <presto.bundle.bootstrap.scope>compile</presto.bundle.bootstrap.scope>
     <presto.bundle.bootstrap.shade.prefix>org.apache.hudi.</presto.bundle.bootstrap.shade.prefix>
+    <shadeSources>true</shadeSources>
```

Comment:
is the indentation right? from the looks of it, does not look ok.

Reply:
I think it is right. Indentation is 4 spaces.